scripts/google/covid_harvard/StatisticalVariable.mcf

```diff
@@ -0,0 +1,27 @@
+Node: dcid:HarvardCOVID19CumulativeCases
+typeOf: dcs:StatisticalVariable
+populationType: dcs:MedicalCondition
```

Comment:
As discussed in today's meeting, let's Positive Cases to be modeled as Positive Tests